packages/tests/tests/peer_exchange.node.spec.ts

```diff
@@ -72,9 +74,10 @@ describe("Peer Exchange", () => {
     });
 
     afterEach(async function () {
-      !!nwaku1 && nwaku1.stop();
-      !!nwaku2 && nwaku2.stop();
-      !!waku && waku.stop().catch((e) => console.log("Waku failed to stop", e));
+      this.timeout(10_000);
+      await nwaku1?.stop();
+      await nwaku2?.stop();
+      await waku?.stop();
```

Comment:
just a note that it wont have a nice print in case of failure
@danisharora099 did you check how it fails? I am curious if it is easily approachable 
usually it is, just not sure about our test setup